src/design-tokens/_widths.yml

```diff
@@ -0,0 +1,8 @@
+aliases:
+  width_30: '30em'
+  width_40: '40em'
+  width_48: '48em'
+  width_64: '64em'
+  width_80: '80em'
+  width_100: '100em'
+  width_120: '120em'
```

Comment:
Just a thought: Would these be more helpful if they were disconnected from the actual width? So like `width_xs`, `width_s`, etc.